integration/scripts/src/runTest.ts

```diff
@@ -0,0 +1,75 @@
+import { ChainlinkNode } from './chainlinkNode'
+import { fetchTests } from './tests'
+import { fetchArgs, fetchCredentials } from './common'
+import * as assert from './asserts'
+
+const main = async () => {
+  const tests = fetchTests()
+
+  const { chainlinkUrl } = fetchArgs()
+  const credentials = fetchCredentials()
+  const node = new ChainlinkNode(chainlinkUrl, credentials)
+
+  const ctx = await assert.context(async (ctx) => {
+    for (let i = 0; i < tests.length; i++) {
```

Comment:
Maybe we can write this in a more "modern" way? For/in, for/of, forEach, map or something similar.